index.html

```diff
@@ -217,6 +221,15 @@ <h2>New Constraints for Captured Display Surfaces</h2>
             indicates a capture capture of a <a>visible display
             surface</a>.</p>
           </dd>
+
+          <dt>ConstrainBoolean muteAudioForSource</dt>
+
+          <dd>
+            <p>A value of <code>true</code> indicates that we mute the local
+            playback for the captured content. For example, when capturing a
+            browser tab, the tab will be muted; when capturing the full screen
+            with audio, then the system audio will be muted.</p>
+          </dd>
```

Comment:
For tab capture, does "The tab will be muted" mean everything is silence, or does this mean we get all system audio except for that tab?

When capturing the full screen, is muting the system audio the same as not requesting audio?

What do you think about this proposal?
https://docs.google.com/document/d/1pgN8Pe2zMaD0tytJVsKmnBT-7gM247YlzzWazm7OwSk/edit?usp=sharing
It overlaps with some of this.